### DISCUSSION TITLE
Keras 3.0 to Browser

### DISCUSSION THREAD

**Discussion by santi-mir** [keras-team/keras#19851] [Q&A] [answered]
Is it possible, in principle, to convert Keras 3.0 models to a format that runs in a web browser (like onnx) ?

I did find [some potential packages](https://pypi.org/project/keras2onnx/) but I wonder if that's what you'd recommend.

**Comment by ibinti** [comment, marked as answer]
I did look the linked package last time when I was trying to find a way, but it seemed outdated and unmaintained.
Then I realized that I could use tf2onnx. It works for me. I do train my model with jax backend, and switch to tensorflow backend when converting to onnx.

**Reply by santi-mir** [reply]
Interesting, I've observed that they have a related step in the roadmap:

https://github.com/keras-team/keras/issues/19519

Indeed the one you refer to is in the onnx repo now, which is a good sign. It's here: https://github.com/onnx/tensorflow-onnx
